### PR TITLE
Timestamp in logs and new output log files for each region

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@flopezag @pratid 
#### Description

Add timestamp formatter to be used as part of logged outputs by nosetests.
New logging configuration.
Create new output logfile for each region with all logged outputs.
#### Testing

Jenkins.

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

File: fiware-region-sanity-tests/.noserc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[nosetests]
+logging-filter=TestCase,novaclient,neutronclient
+logging-level=INFO
+logging-format=%(asctime)s UTC/GMT - %(name)s - %(levelname)s - %(message)s
```
Comment (pratid) [inline fiware-region-sanity-tests/.noserc]: Please remove "GMT", as UTC supersedes that timezone (no matter conversion function is called `time.gmtime`).

File: fiware-region-sanity-tests/resources/logging_sanitychecks.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+# A custom FileHandler is defined by source code when configuring 'TestCase' logger.
+# Following configuration is customized to be loaded from 'fiware_cloud_test_case.setUp'
+
+[sanitychecks_handler_fileHandler]
+level=DEBUG
+filename=./test_logs_{region_name}.log
+
+[sanitychecks_formatter_fileFormatter]
+format=%(asctime)s UTC/GMT - %(name)s - %(levelname)s - %(message)s
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/logging_sanitychecks.conf]: Same as above.

File: fiware-region-sanity-tests/sanity_checks [hunk 3]
```diff
@@ -86,7 +86,7 @@ export OS_TENANT_ID OS_TENANT_NAME OS_USER_DOMAIN_NAME
 
 # Options for nosetests
 TESTS=
-NOSEOPTS="--logging-filter=TestCase,novaclient,neutronclient --logging-level=ERROR"
+NOSEOPTS="--config=.noserc"
```
Comment (pratid) [inline fiware-region-sanity-tests/sanity_checks]: I'd prefer naming the configuration file as `nose.cfg`, given that files `.*rc` are usually reserved from $HOME directories.

**Comment by pratid** [comment]
LGTM

**Comment by jframos** [comment]
This PR fixes #57.

**Pull request merged**